src/chrome/content/rules/AliorBank.pl.xml

```diff
@@ -0,0 +1,160 @@
+<!--
+	Self-signed:
+		- gateway.api
+
+	Connection reset:
+		- static.kantor
+
+	Mismatched:
+		- www.trader
+
+	Refused:
+		- cii
+		- akcept.kantor.de
+		- akcept.m.kantor.de
+		- test.m.kantor.de
+		- www.test.m.kantor.de
+		- akcept.static.kantor.de
+		- test.static.kantor.de
+		- test.kantor.de
+		- www.test.kantor.de
+		- facebook
+		- formularze
+		- www.fuzja
+		- identyfikacja
+		- www.kz
+		- m
+		- malware
+		- mfp
+		- test.mfp
+		- www.test.mfp
+		- najlepszy
+		- native
+		- native1
+		- online
+		- akcept.online
+		- test.online
+		- oszczednosci
+		- fuzja.aliorbank.pl
+		- pomoc-dla-haiti
+		- portale
+		- pozytywy
+		- private-banking
+		- privatebanking
+		- przywileje
+		- sowa
+		- www.sowa
+		- test.das.tmro
+
+	Timeout:
+		- production.developer
+		- emu
+		- ethereum
+		- www.ethereum
+		- fuzja
+		- guestaccess01
+		- guestaccess02
+		- ns1
+		- ns2
+		- online-fuzja
+		- partnerzy
+		- www.schumacher
+		- services
+		- f.video-oddzial
+		- vpn
+		- vpnmobile
+		- wah
+
+	Incomplete certificate chain:
+		- gateway.developer
+		- oauthdemo.developer
+		- gielda
+		- konto-internetowe
+		- link4
+		- pzu
+-->
+<ruleset name="AliorBank.pl">
+	<target host="aliorbank.pl" />
+	<target host="www.aliorbank.pl" />
+	<target host="4x4.aliorbank.pl" />
+	<target host="accelerator.aliorbank.pl" />
+	<target host="analityka.aliorbank.pl" />
+	<target host="ares.aliorbank.pl" />
+	<target host="auth.aliorbank.pl" />
+	<target host="autoryzacja.aliorbank.pl" />
+	<target host="wnioski.bm.aliorbank.pl" />
```

Comment:
`broker.aliorbank.pl`

Reply:
How did you get that domain? Amass didn't find it.

Reply:
search engine

Reply:
Thanks! I usually just use `amass enum -d domain.com --active --brute`.